src/state/electrumX/sagas.ts

```diff
@@ -221,7 +221,7 @@ export function* checkConnection() {
   } catch (e) {
     logger.error('electrumX sagas', `checkConnection error: ${e.message}`);
   } finally {
-    RNBootSplash.hide({ fade: true });
+    RNBootSplash.hide();
```

Comment:
with this, we little change behaviour splash it makes sense?

Reply:
it was me who did this change previously :)